squape/settings.py

```diff
@@ -36,6 +40,9 @@ def _ctx_settings(setting_name, value):
     except Exception:
         raise
     finally:
+        test.log(
```

Comment:
I would prefer to use here report.debug(msg), from squape.report. This would give the user option to decide wheater to see those entries or not.